website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -57,6 +57,51 @@ Use the Accordion when the toggle or the content is relatively complex.
 ![Accordion with complex content](/assets/components/accordion/usage-do-complex.png =770x*)
 !!!
 
+## Sizes
+
+The Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium` for most use cases, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, don't mix different sizes of the AccordionItem.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+The Accordion has two container types: `card` and `flush`, and the `static` property, which applies only to the AccordionItem.
+
+### Card
+
+Use the `card` variant by default to create visual separation between Accordion Sections and other UI elements, organize blocks of content, and distinguish between sections with [Tab](/components/tab). Unlike [Card](/components/card)/Tiles or [Tables](/components/table), the `card` variant groups multiple expandable sections for flexible content management.
+
+!!! Do
+
+![image of DO example for the card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the `flush` variant for tight spaces like within a [Card](/components/card), [Flyout](/components/flyout), `Sidebar`, or other containers where space is limited.
+
+!!! Do
+
+![image of DO example for the Flush variant](/assets/components/accordion/accordion-flush-example.png)
+!!!
+
+### Static AccordionItems
+
+The `isStatic` property in an AccordionItem keeps it fixed in either an open or closed state based on the `isOpen` property. This means users can't toggle it open or closed. It's useful when you want to show important information in your Accordion that should always stay open or closed, making it stand out as different from the other items.
+
+An example of this would be the following:
```

Comment:
[nit] I'd stay away from directional language like "following" and provide a caption that is more indicative of what the actual example is:

```suggestion
This example showcases a Terraform run being triggered with a step that is skipped, using the `isStatic` property to prevent interaction with the toggle of the skipped step.
```